README.md

```diff
@@ -1,45 +1,151 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li>HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML Hint]**
+- **[CSS LINT]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  git clone git@github.com:josman227/Portfolio-Site
+
+### Install
+
+Install this project with: git clone
 
 ### Usage
- - to get a local copy up and running follow these simple example steps.
 
-- Clone this project using the command `git clone https://github.com/josman227/Portfolio-Site`
+To run the project, execute the following command: git clone  
+    
+### Run tests
+
+To run tests, run the following command:
+
+### Deployment
 
-- `Cd feature` folder
+You can deploy this project using:
 
-- Open `index.html` in your local browser or using Live Server in Visual Studio Code.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
-## Authors
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
 
 👤 **Josiah**
 
 - GitHub: [@josman227](https://github.com/josman227)
-- Twitter: [@josiahmutinhiri](https://twitter.com/@josiahmutinhiri)
-- LinkedIn: [josiahmutinhiri](https://linkedin.com/in/josiahmutinhiri)
+- Twitter: [@josiahmutinhiri](https://twitter.com/josman227)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/josiahmutinhiri)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Contacts]**
+- [ ] **[Projects]**
+- [ ] **[Feed]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 
+## ⭐️ Show your support <a name="support"></a>
 
-## 🤝 Contributing
+If you like this follow me and feature
 
-Contributions, issues, feature requests are welcome!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- ACKNOWLEDGEMENTS -->
 
-## Show your support
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-Give a ⭐️ if you like this project!
+I would like to thank Microverse Reveiwers
+    
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- Thanks to Github 
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./LICENSE) licensed.
 
-This project is [MIT](.md) licensed.
```

Comment:
Unfortunately your link to the license file is broken. Please make sure you have a working link to your license file. 👍